ls1intum/Artemis
# bugfix/fix assessment count

### Checklist

### Motivation and Context


The assessment counts for programming exercises with manual assessments aren't calculated correctly.

### Description

Similar to the submission counts, assessment counts are not calculated separately for programming exercises.

### Steps for Testing


1. Log in to Artemis
2. Create a programming exercise with manual tutor assessments
3. Submit some solutions as student(s)
4. Grade the submissions as a tutor
5. Check (tutor|instructor)-(course|exercise)-dashboard to contain the correct statistics and numbers
